src/applications/pensions/components/FormAlerts/index.jsx

```diff
@@ -163,3 +163,23 @@ export const AssetTransferFormAlert = () => (
     </p>
   </va-alert>
 );
+
+export const LandMarketableBox = () => (
```

Comment:
Can we rename this 'LandMarketableAlert' to remain inline with the other naming conventions in this file?
```suggestion
export const LandMarketableAlert = () => (
```